python/cloud-run-python-hello-world/app.py

```diff
@@ -0,0 +1,45 @@
+"""
+A sample Hello World server.
+"""
+import os
+import requests
+
+from flask import Flask, render_template
+
+# pylint: disable=C0103
+app = Flask(__name__)
+
+
+def get_metadata(item_name):
+    metadata_url = 'http://metadata.google.internal/computeMetadata/v1/'
+    headers = {'Metadata-Flavor': 'Google'}
+
+    try:
+        r = requests.get(metadata_url + item_name, headers=headers)
+        return r.text
+    except:
+        return 'Unavailable'
```

Comment:
Could we refine it so that it distinguishes between running locally vs running on Cloud Run and not being able to detect the project ID?
If it runs locally, we should either print the project ID set in local gcloud config or at least print a note that makes it clear that it's not available due to running locally.

Reply:
Let's standardize on using GOOGLE_CLOUD_PROJECT env var to preempt the use of metadata server. Not quite local detection but meaningful as an override. The Go PR includes this.